mathics_scanner/data/named-characters.yml

```diff
@@ -4469,15 +4469,14 @@ Infinity:
   wl-unicode-name: INFINITY
 
 # Infix isn't really an operator
-# See also Tilde
+# See also RawTilde
 Infix:
   ascii: "~"
   amslatex: "\\textasciitilde"
   has-unicode-inverse: false
   is-letter-like: false
 # operator-name: Infix
   unicode-equivalent: "~"
```

Comment:
Something for another time. The comments at the top say that if unicode-equivalent is the same as ascii, please omit.